XIVSlothCombo/Combos/PvE/SCH.cs

```diff
@@ -445,8 +445,7 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                     // Lucid Dreaming
                     if (IsEnabled(CustomComboPreset.SCH_Succor_Combo_Lucid) &&
                         ActionReady(All.LucidDreaming) &&
-                        LocalPlayer.CurrentMp <= Config.SCH_Succor_Combo_LucidOption &&
-                        CanSpellWeave(actionID))
+                        LocalPlayer.CurrentMp < 1000)
```

Comment:
Just noting for myself that this is an edge-case prevention on the off-chance the user doesn't have enough MP to cast `Succor` in the first place